### DISCUSSION TITLE
fish-speech-1.5-yth-lora

### DISCUSSION THREAD

**Discussion by rhfu** [fishaudio/fish-speech#781] [General]
python tools/llama/merge_lora.py
--lora-config r_8_alpha_16
--base-weight checkpoints/fish-speech-1.5
--lora-weight results/$project/checkpoints/step_000000010.ckpt
--output checkpoints/fish-speech-1.5-yth-lora/
﻿
After fine-tuning, how is the directory generated by checkpoints/fish-speech-1.5-yth-lora used in inference? Do you want to specify it during inference?

**Comment by Emersonksc** [comment]
python -m tools.webui \
    --llama-checkpoint-path "checkpoints/fish-speech-1.4" \
    --decoder-checkpoint-path "checkpoints/fish-speech-1.4/firefly-gan-vq-fsq-8x1024-21hz-generator.pth" \
    --decoder-config-name firefly_gan_vq \
    --compile
    this is used for inferencing, and --llama-checkpoint-path is where your output directory should go, for your example: --llama-checkpoint-path "checkpoints/fish-speech-1.5-yth-lora/"

**Comment by cod3r0k** [comment]
I hear a beep sound after training. Why is this happening? After some iterations (about 2 days) during epoch 0, the process suddenly finished.
I checked the logs, and it just returned a beep sound. Why is this happening? The accuracy in the logs is 46%.
